### DISCUSSION TITLE
esp8266 not working

### DISCUSSION THREAD

**Discussion by robgt978** [syssi/esphome-smg-ii#50] [Q&A]
![interfaccia](https://github.com/syssi/esphome-smg-ii/assets/152035751/4754394d-c9c2-4a21-abc8-058af8b0a7d4)
![errore](https://github.com/syssi/esphome-smg-ii/assets/152035751/59f4278f-ca70-4f0e-84cb-d92af8427bd7)
![codice1](https://github.com/syssi/esphome-smg-ii/assets/152035751/78e0ce02-38bb-4d90-918b-088745a7468e)
![codice2](https://github.com/syssi/esphome-smg-ii/assets/152035751/b80f6ef8-3ddb-43f1-93f1-15e4a9e6c89c)

I'm using esp8266 d1mini I've tried 1000 times even with mqtt server but I don't receive data from the inverter. help me thanks
the white/orange is pi1 from RJ45 and orange is pin2

**Comment by syssi** [comment]
Could you provide a close-up photo of your RS232 converter? Please make sure the converter is equipped with a MAX3232 (the 3.3V if the MAX232 converter chip). How many LEDs do you see on the converter PCB? One power LED or one RX and one TX LED?

**Reply by robgt978** [reply]
Only One led Red. I have tested the interface with PIN 2-3 connected between 
![Screenshot_2023-12-06-16-19-19-89_b5f6883d2c20a96c53babc0b4ac88108](https://github.com/syssi/esphome-smg-ii/assets/152035751/efd69ea6-12f3-4b15-8552-5d0b0463e5be)
 and a software for loopback

**Reply by syssi** [reply]
Is the loopback test working?
